src/styles/components/expanding-table/styles.less

```diff
@@ -85,6 +85,10 @@
 
   .expanding-table-child {
 
+    > div {
```

Comment:
I didn't quite realize that you've been discussing that solution. I ok merging it, thanks for trying out different solutions here 💪